src/corrgi/dask.py

```diff
@@ -124,86 +126,17 @@ def count_cross_pairs(
        right_pix (HealpixPixel): The pixel corresponding to `right_df`.
        left_catalog_info (CatalogInfo): The left catalog metadata.
        right_catalog_info (CatalogInfo): The right catalog metadata.
-       bins (np.ndarray): The separation bins, in angular space.
-       params (Munch): The gundam subroutine parameters.
+       correlation (Correlation): The correlation instance.
 
     Returns:
        The count histogram for the partition pair.
     """
     try:
-        return _count_cross_pairs(
+        return correlation.count_cross_pairs(
             left_df,
             right_df,
             left_catalog_info,
             right_catalog_info,
-            bins,
-            params,
         )
     except Exception as exception:
         dask_print(exception)
```

Comment:
Is printing the exception ok here? You don't need to raise it and it's ok for the program to continue?

Reply:
Great catch, we should be printing the exception and raising it right after!